openstf/pipeline/train_model_sklearn.py

```diff
@@ -20,11 +19,8 @@
 OLD_MODEL_PATH = '.'
```

Comment:
I think most code uses double quotes for strings, probably good to stick to one convention.